device-types/Cisco/AIR-CAP2602E-T-K9.yaml

```diff
@@ -0,0 +1,22 @@
+---
+manufacturer: Cisco
+model: Aironet 2602E-T
+slug: cisco-air-cap2602e-t-k9
+part_number: AIR-CAP2602E-T-K9
+u_height: 0
+is_full_depth: false
+weight: 2.3
+weight_unit: lb
+console-ports:
+  - name: con0
+    type: rj-45
+interfaces:
+  - name: WiFi
+    type: ieee802.11n
+  - name: eth0
```

Comment:
`eth0` probably isnt the correct config name for this interface. Do you know what the proper config name is for this interface?

Reply:
GigabitEthernet0 (sometimes GigabitEthernet1 for the second port) is the correct naming convention for Cisco WAPs

Reply:
@DanSheps I saw these have ports labelled AUX, are those GigabitEthernet1?

Reply:
Aux is more of a "modem" port if I am remembering.  It could be modelled as a RJ45 console port.